tests/plugins/selectall/manual/selectall.html

```diff
@@ -0,0 +1,15 @@
+<h2>Classic editor</h2>
+
+<div id="editor1">
+	<p>Some text</p>
+
+	<p>Some text2</p>
+</div>
+
+<script>
+	if ( ( CKEDITOR.env.ie && CKEDITOR.env.version < 11 ) || bender.tools.env.mobile ) {
```

Comment:
As this is only Edge issue we decided that test should be ignored for other browsers.